app/client/perf/src/ci/supabase.js

```diff
@@ -99,7 +99,7 @@ const saveData = async (results) => {
       row["meta"] = run_meta.id;
       const runs = results[action][metric];
       runs.forEach((value, i) => {
-        row["value"] = value;
+        row = { ...row, value };
         rows.push(row);
```

Comment:
```suggestion
        rows.push(row);
```
```suggestion
        rows.push({ ...row, value });
```
This could have also worked fine here.